freetype/build.zig

```diff
@@ -48,6 +48,12 @@ pub fn build(b: *std.build.Builder) !void {
     const mode = b.standardReleaseOptions();
     const target = b.standardTargetOptions(.{});
 
+    const test_app = b.addStaticLibrary("test_app", null);
+    test_app.setBuildMode(mode);
+    test_app.setTarget(target);
+    link(b, test_app, .{ .freetype = .{ .brotli = true } });
+    test_app.install();
```

Comment:
I've been making an effort to not create unused static libraries (which add to compile times), and not to install them for the reasons described in https://github.com/hexops/mach/issues/232

This change and the GLFW change below go against that. But why? How's it helpful?

Reply:
because `zig build` need to do something. otherwise e.g cross-compiling in CI is pointless

Reply:
OK, I see what you're going for here. But it doesn't seem like we should `.install()` these, as they are for testing only after all.

Separately, I want to ask: does `zig build` not build the test binary themselves? It seems to me we should be able to have `zig build` build the tests.. and those should build for the target OS, too - and then we wouldn't need to build useless static libraries like this.

Reply:
(should address this before merging)

Reply:
there's `b.testExe`

Reply:
`addTestExe` looks good to me!

Reply:
https://github.com/hexops/mach/pull/429/files#r928133589